src/PrestaShopBundle/EventListener/TokenizedUrlsListener.php

```diff
@@ -69,6 +71,10 @@ public function onKernelRequest(GetResponseEvent $event)
     {
         $request = $event->getRequest();
 
+        if (TokenInUrls::isEnabled()) {
+            return;
```

Comment:
I would expect the opposite from this class name. Would `NoTokenInUrl` better suit what you implement in this PR?